docs/src/main/sphinx/release/release-450.md

```diff
@@ -0,0 +1,131 @@
+# Release 450 (19 Jun 2024)
+
+## General
+
+* Add support for defining a number of rows to be pushed from a filter into
+  values with the `optimizer.push-filter-into-values-max-row-count`
+  configuration property. ({issue}`22105`)
+* Add support for specifying an Azure blob endpoint for accessing spooling in
+  fault-tolerant execution with the `exchange.azure.endpoint` configuration
+  property. ({issue}`22218`)
+* Expose driver execution statistics via JMX. ({issue}`22427`)
+* Improve performance of the {func}`first_value` and {func}`last_value`
+  functions. ({issue}`22092`)
+* Improve performance for large clusters under heavy workloads. ({issue}`22039`)
+* Improve performance of queries with simple predicates. This optimization can
+  be disabled using the `experimental.columnar-filter-evaluation.enabled`
+  configuration property or the `columnar_filter_evaluation_enabled` session
+  property. ({issue}`21375`)
+* {{breaking}} Improve performance of distinct aggregations, and replace the
+  `optimizer.mark-distinct-strategy` and
+  `optimizer.optimize-mixed-distinct-aggregations` configuration properties with
+  the new `optimizer.distinct-aggregations-strategy` property. ({issue}`21907`)
+* Improve performance of reading JSON files. ({issue}`22348`)
+* Improve performance for the {func}`date_trunc`, {func}`date_add`, and
+  {func}`date_diff` functions. ({issue}`22192`)
+* Fix failure when loading the [](/admin/event-listeners-openlineage). ({issue}`22228`)
+* Fix potential incorrect results when a cached value was removed or updated. ({issue}`22285`)
+
+## Security
+
+* Add support for exclusively using web identity for authentication when using
```

Comment:
This item is hard to parse and understand. How about:

"Add support for using web identity exclusively for authentication when running on Amazon EKS with the legacy S3 file system enabled. This can be configured via the `trino.s3.use-web-identity-token-credentials-provider` property."